test/trustedresources.go

```diff
@@ -124,44 +115,184 @@ func SetupTrustedResourceConfig(ctx context.Context, keypath string, resourceVer
 	return store.ToContext(ctx)
 }
 
-// GetSignerFromFile generates key files to tmpdir, return signer and pubkey path
-func GetSignerFromFile(ctx context.Context, t *testing.T) (signature.Signer, string, error) {
+// SetupVerificationPolicies set verification policies
+func SetupVerificationPolicies(t *testing.T) (signature.SignerVerifier, *fakek8s.Clientset, []*v1alpha1.VerificationPolicy) {
 	t.Helper()
+	sv, _, pub, err := GenerateKeys(pass("1234"))
+	if err != nil {
+		t.Fatal(err)
```

Comment:
in general when handling errors it is useful to add context such as 'could not generate key ... ' vs just passing the error along without context (more on this in our standards at https://github.com/tektoncd/community/blob/main/standards.md#code)